.dockerignore

```diff
@@ -0,0 +1,26 @@
+# git stuff
+.git
+.github
+
+# The path for kubevirtci installation
+_kubevirtci
+
+# The old path for kubevirtci installation, in case checkout is done from and
+# old branch this can exists
+kubevirtci
+
+# No need for source code is already compiled
```

Comment:
If you don't care for so many directories, maybe you can just selectively `COPY` exactly what you need instead of copying the whole project and then blacklisting here?

Reply:
Issue is not about  `COPY` issue is about where do you run `docker build [dir]` and that's done by `operator-sdk` for example for handler we don't need the manifests so doing `docker build build/ -t ...` would be enough, but that's not the case for operator that need the manifests

Reply:
Then please add a huge comment on the top of this file saying it is only for operator-sdk due to this and that, so we can drop it once the situation with operator-sdk gets better.